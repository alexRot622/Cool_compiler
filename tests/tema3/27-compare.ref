Loaded: trap.handler.nogc
The following symbols are undefined:
main

OK
OK
failed
failed
failed
OK
COOL program successfully executed
