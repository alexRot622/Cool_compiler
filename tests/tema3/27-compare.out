Loaded: trap.handler.nogc
The following symbols are undefined:
main

OK
OK
OK
OK
OK
OK
COOL program successfully executed
